src/steamship/cli/cli.py

```diff
@@ -65,6 +69,78 @@ def ships():
     click.secho()
 
 
+@click.command()
+@click.option(
+    "--port",
+    "-p",
+    type=int,
+    default=8080,
+    help="Port to host the server on.",
+)
+@click.option(
+    "--invocable_handle",
+    "-i",
+    type=str,
+    default=None,
+    help="Handle of the package or plugin being hosted.",
+)
+@click.option(
+    "--invocable_version_handle",
+    "-v",
+    type=str,
+    default=None,
+    help="Handle of the package or plugin version being hosted.",
+)
+@click.option(
+    "--invocable_instance_handle",
+    "-h",
+    type=str,
+    default=None,
+    help="Handle of the package or plugin instance being hosted.",
+)
+@click.option(
+    "--api_key",
+    "-k",
+    type=str,
+    default=None,
+    help="API Key to hard-code for hosting.",
+)
+def serve(
+    port: int = 8080,
+    invocable_handle: Optional[str] = None,
+    invocable_version_handle: Optional[str] = None,
+    invocable_instance_handle: Optional[str] = None,
+    api_key: Optional[str] = None,
+):
+    """Serve the local invocable"""
+    initialize()
+    path = find_api_py()
+    api_module = get_api_module(path)
+    invocable_class = get_class_from_module(api_module)
+    click.secho(f"Found Invocable: {invocable_class.__name__}")
+
+    server = SteamshipHTTPServer(
+        invocable_class,
+        port=port,
+        invocable_handle=invocable_handle,
+        invocable_version_handle=invocable_version_handle,
+        invocable_instance_handle=invocable_instance_handle,
```

Comment:
Would it make more sense to default these from the manifest rather than `None`?  I can't think of a reason it would matter to the debugging flow though